### DISCUSSION TITLE
TShock update

### DISCUSSION THREAD

**Discussion by Cailleur** [Pryaxis/TShock#2259] [Q&A]
With the newest Terraria update being workshop support and not bringing in any new content, how fast will the team roll out a new compatible version? Also, is there a way I can run my server on the newest version of Terraria but use an outdated version of TShock?

**Comment by ghost** [comment]
same thing my friend bought server and now we can't play because of useless terraria update

**Comment by moisterrific** [comment]
nobody knows how long it will take cuz it depends on a lot of factors. And no, trying to run older TShock w/ newer client (and vice versa) will not work

**Comment by dragoonzwei** [comment]
As an option, you can always roll back your client version on Steam.  There are several guides available (on Steam) on how to do so.

**Reply by Thunder1723** [reply]
Where are the guides you speak of?  The ones I have found are outdated and won't work with the previous version of Terraria.

**Reply by dragoonzwei** [reply]
You should be able to find it if you do a web search for "Steam How to Downgrade or Change the Version of a Game".  That is the guide I used.  The only problem I have sometimes is the game reverting to newer release, but I just replace it again with the old version.
